src/harness/reference_models/geo/README.md

```diff
@@ -9,7 +9,7 @@ models:
  - `vincenty.py`: precise methods for deriving geodesic between two points in the earth
  modeled as ellipsoid
  - `utils.py`: utility routines for computing polygon area, gridding a polygon, etc..
- - `census_tract.py`: Census tract driver to read JSON census tracts geometries.
+ - `county.py`: county driver to read JSON coutnies geometries.
```

Comment:
typo "Coutnies"

Reply:
thanks Serge!
I already made the PR with the changes in Common-data, please feel free to check ;)
for your comment about the PR in Common-data ("you can keep both driver code and keep the old data"), did you mean census tracts data&scripts?
I don't know if we need to keep census tracts related data as they exist in the Github history.

Reply:
I was suggesting to keep the census tract data and reading driver (within Common-Data repo), since it is located in a different repo that is supposed to be of use for all Winnforum projects. SAS is one, 6GHz-AFC is another, ...
Why not let them since they are already there and perhaps of use for one of these future projects